infrahub_sdk/node.py

```diff
@@ -678,6 +690,11 @@ def __init__(self, schema: MainSchemaTypesAPI, branch: str, data: dict | None =
         self._branch = branch
         self._existing: bool = True
 
+        # Generate a unique ID only to be used inside the SDK
+        # The format if this ID is purposely different from the ID used by the API
+        # This is done to avoid confusion and potential conflicts between the IDs
+        self._internal_id = shortuuid.uuid()
```

Comment:
Instead of installing an extra package just for this can we add a function such as:

```python
import uuid
import base64

def short_id() -> str:
    return base64.urlsafe_b64encode(uuid.uuid4().bytes).rstrip(b'=').decode('ascii')
```